### ISSUE TITLE
Recent release with no code change

### ISSUE THREAD

**Issue by s7726** [rivafarabi/deckboard#392]
What's up with the release in April, when there have been no code changes in three months?

**Comment by Yurlyn** [comment]
It was a hotfix plus something did change. You can now select the behavior for sound buttons, either play from start or start/stop.

**Comment by s7726** [comment]
Just questioned since the diff appeared to be empty.

**Comment by YaFou** [comment]
> 
> 
> What's up with the release in April, when there have been no code changes in three months?

The software is not an open-source project. As a consequence, the code isn't here in this repository. This repository is using for the website and to put release files.

**Comment by Yurlyn** [comment]
Yeah, I restarted it and then it updated but I wouldn't have known what the update was if I didn't check here xD
